ocs_ci/templates/workloads/helper_scripts/check_xattr.py

```
@@ -0,0 +1,137 @@
+    max_workers = min(max_workers, 100, file_count)
+
+    # Example usage
+    files = create_files("testfile_", file_count, directory)
+    log.info(f"Creation of {len(files)} files in '{directory}' directory completed.")
+
+    # Toggle author xattr for each file continuously in parallel batches
+    try:
+        with ThreadPoolExecutor(max_workers=max_workers) as executor:
+            while True:
+                futures = [executor.submit(toggle_author_xattr, file) for file in files]
+                for file, future in zip(files, futures):
```

Ruff (0.15.12)

[warning] Loop control variable `file` not used within loop body

Rename unused `file` to `_file`

(B007)